src/layout/Modal.tsx

```diff
@@ -23,6 +23,7 @@ export function Modal(props: ModalProps) {
           position: 'absolute',
           top: 0,
           left: 0,
+          zIndex: 90,
```

Comment:
I think we should use a portal instead (add the modal at the end of the DOM so it's in front of everything)